assets/scripts/users/authentication.js

```diff
@@ -70,7 +70,7 @@ function saveSignInDataLocally () {
   if (signInData) {
     window.localStorage[LOCAL_STORAGE_SIGN_IN_ID] = JSON.stringify(signInData)
```

Comment:
Is `saveSignInDataLocally()` called anywhere else?

If so, let's also set this item correctly